.travis.yml

```diff
@@ -21,70 +21,123 @@ before_install:
+  - sudo apt-get install upx-ucl -y
+  - sudo apt-get install rsync -y
   - rsync -az ${TRAVIS_BUILD_DIR}/ $HOME/gopath/src/github.com/kubeedge/kubeedge/
   - export TRAVIS_BUILD_DIR=$HOME/gopath/src/github.com/kubeedge/kubeedgee
   - go get github.com/onsi/ginkgo/ginkgo
   - export GO111MODULE=on
   - export GOFLAGS=-mod=vendor
   - cd $HOME/gopath/src/github.com/kubeedge/kubeedge
-  - sudo apt-get install build-essential
-  - sudo apt-get install gcc-arm-linux-gnueabi
-  - sudo apt-get install upx-ucl
 
 install: true
 
 jobs:
   include:
     - stage: "Verify vendor"
+      arch: amd64
       script: make verify
-      name: "check if the vendor is up to date"
+      name: "check if the vendor is up to date on amd64"
+    - script: make verify
+      arch: arm64
+      name: "check if the vendor is up to date on arm64"
 
     - stage: "Test lint"
       script: make cloud_lint
-      name: "cloud_lint"     # names the eighth Tests stage job
+      name: "cloud_lint"
     - script: make bluetoothdevice_lint
       name: "bluetooth_device_lint"
     - script: make keadm_lint
       name: "keadm_lint"
     - script: make edge_lint
       name: "edge_lint"
 
-    - stage: "Test Build"                # naming the Tests stage
-      script: make 
-      name: "builds cloud and edge components"            # names the first Tests stage job
+    - stage: "Test Build"
+      script: make
+      arch: amd64
+      name: "builds cloud and edge components on amd64"
+    - script: make
+      arch: arm64
+      name: "builds cloud and edge components on arm64"
     - script: make bluetoothdevice
-      name: "build bluetooth_device"
+      arch: amd64
+      name: "build bluetooth_device on amd64"
+    - script: make bluetoothdevice
+      arch: arm64
+      name: "build bluetooth_device on arm64"
+    - script: make edge_small_build
+      arch: amd64
+      name: "build small edge version on amd64"
     - script: make edge_small_build
-      name: "build small edge version"
+      arch: arm64
+      name: "build small edge version on arm64"
     - script: make edge_cross_build
-      name: "cross build edge"     # names the fifth Tests stage job
+      name: "cross build edge"
     - script: make edge_cross_build_v7
-      name: "cross build edge for armv7"     # names the fifth Tests stage job
+      name: "cross build edge for armv7"
     - script: make edgesite_cross_build
-      name: "cross build edgesite"    # names the fifth Tests stage job
+      name: "cross build edgesite"
     - script: make edgesite_cross_build_v7
-      name: "cross build edgeitefor armv7"     # names the fifth Tests stage job
+      name: "cross build edgeite for armv7"
 
     - stage: "unit tests"
       script: make edge_test
-      name: "unit test edge"     # names the third Tests stage job
+      arch: amd64
+      name: "unit test edge on amd64"
+      #- script: make edge_test
+      #- arch: arm64
+      #- name: "unit test edge on arm64"
```

Comment:
please remove these comments

Reply:
this going to be unmasked in the follow on PR along with the fix of the failure.